src/current/v23.1/migration-overview.md

```diff
@@ -5,87 +5,304 @@ toc: true
 
-## Step 3. Test and update your application
+The following is a high-level overview of the migration steps. {% comment %}For details on this migration strategy, see [Migration Strategy: Zero Downtime](migration-strategy-lift-and-shift.html).{% endcomment %}
 
-As the final step of migration, you will likely need to make changes to how your application interacts with the database. For example, refer to [features that differ from PostgreSQL](postgresql-compatibility.html#features-that-differ-from-postgresql).
+To prioritize consistency and minimize downtime:
 
-Unless you changed the integer size when [migrating the schema](#schema-design-best-practices), your application should also be written to handle 64-bit integers. For more information, see [Considerations for 64-bit signed integers](int.html#considerations-for-64-bit-signed-integers).
+1. {% include {{ page.version.version }}/migration/load-data-third-party.md %} Select the tool's option to **replicate ongoing changes** after performing the initial load of data onto CockroachDB. 
+1. As the data is migrating, you can use [`molt verify`](https://github.com/cockroachdb/molt) to validate the consistency of the data between the legacy database and CockroachDB.
+1. Once nearly all data from your source database has been moved to CockroachDB (for example, with a <1 second delay or <1000 rows), stop application traffic to your source database. **This begins downtime.**
+1. Wait for replication to CockroachDB to complete.
+1. Perform a [cutover](#cutover-strategy) by resuming application traffic, now to CockroachDB.
 
-We **strongly recommend testing your application against CockroachDB** to ensure that:
+To achieve zero downtime with inconsistency:
 
-1. The state of your data is what you expect post-migration.
-1. Performance is sufficient for your application's workloads. Follow the [SQL Performance Best Practices](performance-best-practices-overview.html) and implement [transaction retry logic](transactions.html#transaction-retries).
+1. {% include {{ page.version.version }}/migration/load-data-third-party.md %} Select the tool's option to replicate ongoing changes after performing the initial load of data onto CockroachDB.
+1. As the data is migrating, you can use [`molt verify`](https://github.com/cockroachdb/molt) to validate the consistency of the data between the legacy database and CockroachDB.
+1. Once nearly all data from your source database has been moved to CockroachDB (for example, with a <1 second delay or <1000 rows), perform a [cutover](#cutover-strategy) by pointing application traffic to CockroachDB.
+1. Manually reconcile any inconsistencies caused by writes that were not replicated during the cutover.
+1. Close the connection to the legacy database when you are ready to finish the migration.
 
```

Comment:
I still think we should include a "complete the migration" step that discusses notifying the team, retiring the test environments, retiring the migration tooling, etc.